src/webgpu/api/validation/encoding/queries/general.spec.ts

```diff
@@ -27,6 +27,26 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
+g.test('occlusion_query,invalid_query_set')
+  .desc(
+    `
+Tests that set occlusion query set with all types in render pass descriptor:
+- type {occlusion (control case), pipeline statistics, timestamp}
+- invalid query set that failed during creation
```

Comment:
This case would probably be easier as a separate test which has just 2 cases (valid and invalid).